app/Platform/src/Models/MemberProfile.php

```diff
@@ -2,22 +2,37 @@
 
 namespace App\Platform\Models;
 
+use App\Framework\Models\User;
 use App\Platform\Database\factories\MemberProfileFactory;
 use Illuminate\Database\Eloquent\Factories\Factory;
 use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
 use Illuminate\Database\Eloquent\SoftDeletes;
 
 class MemberProfile extends Model
 {
     use HasFactory;
     use SoftDeletes;
 
+    protected $fillable = [
```

Comment:
I don't think we need these, check if the model is unguarded in the framework's AppServiceProvider